lib/Db/FederatedInviteMapper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Contacts\Db;
+
+use OCA\CloudFederationAPI\Db\FederatedInviteMapper as DbFederatedInviteMapper;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @template-extends QBMapper<FederatedInvite>
+ */
+class FederatedInviteMapper extends DbFederatedInviteMapper {
```

Comment:
not a huge fan of this,
if the DB is created and managed from cloud_federation_api, could this request be also integrated to the app, and calling an API from the cloud_federation_api to select/update/delete data from/to database ?